ArchiSteamFarm/Steam/Interaction/Commands.cs

```diff
@@ -499,12 +516,12 @@ public sealed class Commands {
 		return gamesOwned;
 	}
 
-	private async Task<string?> Response2FA(ulong steamID) {
-		if ((steamID == 0) || !new SteamID(steamID).IsIndividualAccount) {
-			throw new ArgumentOutOfRangeException(nameof(steamID));
+	private async Task<string?> Response2FA(EAccess access) {
+		if (!Enum.IsDefined(typeof(EAccess), access)) {
```

Comment:
These are all `private` methods only being called from within `Task<string?> Response(EAccess, string, ulong = 0)`. While it may be nice (and smart) to check this here, the check **theoretically** is redundant and does not accomplish anything. I'm not saying to remove it, but rather suggesting to think about whether it is really necessary.

Reply:
Yes, this is a safeguard against invalid usage, not against public callers but our own code as well. In unlikely case of me forgetting to handle it properly earlier, the intention of every method is to throw on invalid input which already happened and helped ASF in the past.

Every method should carry a safeguard, regardless of its visibility.